<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabMain">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>1</y>
     <width>691</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Nuevo Usuario</string>
    </attribute>
    <widget class="QLineEdit" name="txtName">
     <property name="geometry">
      <rect>
       <x>186</x>
       <y>90</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>95</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre Completo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDNI">
     <property name="geometry">
      <rect>
       <x>186</x>
       <y>147</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblDNI">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cédula</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPhone">
     <property name="geometry">
      <rect>
       <x>186</x>
       <y>207</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblPhone">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Celular</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAddres">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>267</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblAddres">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>270</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Dirección</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblBirth">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>325</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha de Nacimiento</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtBirth">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>322</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>dd/mm/aaaa</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTitle">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>REGISTRAR NUEVO USUARIO</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>54</y>
       <width>691</width>
       <height>1</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSaveUser">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>420</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblBanner">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>510</y>
       <width>681</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>510</y>
       <width>691</width>
       <height>1</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Registrar Compra</string>
    </attribute>
    <widget class="QLineEdit" name="txtDNIRegister">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblDNIRegister">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>26</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cédula</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
